packages/global-cli/index.js

```diff
@@ -36,13 +36,12 @@
 // /!\ DO NOT MODIFY THIS FILE /!\
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-const fs = require('fs');
-const path = require('path');
-const exec = require('child_process').exec;
-const execSync = require('child_process').execSync;
-const chalk = require('chalk');
-const prompt = require('prompt');
-const semver = require('semver');
+import fs from 'fs';
+import path from 'path';
+import child_process from 'child_process';
+import chalk from 'chalk';
+import prompt from 'prompt';
+import semver from 'semver';
```

Comment:
`global-cli` is not compiled with Babel, please revert changes here for now.

Reply:
And it's going to be deprecated anyway, so I'll just skip it.